crates/matrix-sdk-crypto/src/olm/group_sessions/inbound.rs

```diff
@@ -496,9 +522,8 @@ pub struct PickledInboundGroupSession {
     pub signing_key: SigningKeys<DeviceKeyAlgorithm>,
     /// The id of the room that the session is used in.
     pub room_id: OwnedRoomId,
-    /// Flag remembering if the session was directly sent to us by the sender
-    /// or if it was imported.
-    pub imported: bool,
+    /// Where we obtained the group session from.
+    pub key_source: KeySource,
```

Comment:
How do we migrate data?

Reply:
I've implemented migration for here.  How do we migrate in the FFI part?  Do we just replace `imported` with `key_source` in `PickledInboundGroupSession` and leave it up to the app to migrate?